components/TextEditor/TextEditorContainer.module.scss

```diff
@@ -2,6 +2,7 @@
 .editorBox {
   border-style: solid;
   border-width: 5px;
+  width: fit-content;
```

Comment:
Does this just make the containing box smaller?
If so, it feels like that's the wrong way around - can't we make the editor bigger to use all the available space?

Reply:
Doing so leads to the return of the empty space: 
![image](https://user-images.githubusercontent.com/47330108/80593526-4d110d00-8a19-11ea-8910-1f699817c1f8.png)
Think it stretches the editor